### ISSUE TITLE
Display total size of current directory

### ISSUE THREAD

**Issue by fritzrehde** [kamiyaa/joshuto#229]
I would like to somehow display the total size of a directory's content (like joshuto does with files). 
Out of curiousity, is there a reason this isn't implemented directly into joshuto already? The same feature exists for files. The argument against it is probably the performance overhead (I imagine it would be terribly slow for deeply nested directories). Maybe this could be added as an option that can be toggled on or off with a keybinding (I could look into it). You're in a parent directory of directories you want to know the size of, toggle the option, see the size, and toggle back off when you want performance again.
If you don't want this implemented directly into joshuto: The command `du -sh "$DIRECTORY"` would easliy let me do that for a single directory. But is there a way to manually (through a keybinding) call this command on a line in joshuto and somehow display the output somewhere in joshuto?

**Comment by krivahtoo** [comment]
@kamiyaa What do you think of adding this by calulating the actual size of the directory contents instead of counting entries in `JoshutoMetadata::_directory_size`?

We don't have to completely remove count, there could be an option to toggle between count and size.

Here is a preview of what I was able to put together.
![2023-05-05-194153_663x405_scrot](https://user-images.githubusercontent.com/41364823/236517190-57d54a8e-36eb-4e45-9c5b-c861e8042242.png)

**Comment by DLFW** [comment]
Is this recursive?

**Comment by krivahtoo** [comment]
No, it only calculates a single depth and updates lazily as you navigate.
